src/app/api/boosts/claim/route.ts

```diff
@@ -0,0 +1,116 @@
+import { PrismaClient } from '@prisma/client';
+import { NextResponse, type NextRequest } from 'next/server';
+import '@/utils/helper';
+import { Network, Alchemy } from "alchemy-sdk";
+
+const settings = {
+    apiKey: process.env.ALCHEMY_ID,
+    network: Network.BASE_MAINNET,
+};
+
+const alchemy = new Alchemy(settings);
```

Comment:
doesn't this mean alchemy will be initialized every time we call /claim endpoint?
I think this should be initialized in Providers.tsx?